docx/enum/table.py

```diff
@@ -69,3 +69,35 @@ class WD_TABLE_DIRECTION(Enumeration):
             'in the rightmost position.'
         ),
     )
+
+
+class WD_VERTICAL_ALIGNMENT(XmlEnumeration):
```

Comment:
It looks like this enumeration can also be used by sections and perhaps shapes. So we should create a new module named 'shared.py' in this same directory and place this enumeration there.

----

We'll need the documentation page for this as well. I got tired of writing those by hand so you'll find a property on the enumeration named `.__docs_rst__` that generates the exact text of the enumeration documentation page. It goes under /docs/api/enum I believe and the filename is the same as the __ms_name__ below. You'll need to add a line for the new page to the index.rst page in that same directory.

```python
>>> from docx.enum.shared import WD_ALIGN_VERTICAL as wd
>>> print(wd.__docs_rst__)
# then copy and paste or whatever :)
```